### PR TITLE
documentation improvements

### PR DESCRIPTION
**Pull request by schmalliso**
- uses XHGui consistently throughout when referring to XHGui
- streamlines the installation so that the steps are all steps
- rephrases for easier reading
- typo fixes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,84 +4,89 @@ xhgui
...
-* Set the permissions on the `cache` cache directory to allow the webserver to
-  create files.  If you're lazy `0777` will work. Run:
+   ```
+   chmod -R 0777 cache
+   ```
 
-  ```
-  chmod -R 0777 cache
-  ```
+4. Start a MongoDB instance. XHGui uses the MongoDB instance to store
```
Comment (schmalliso) [inline README.md]: This wasn't really alluded to previously, so I added it explicitly before talking about configuring to connect to the `mongod` / setting up indexes.

File: README.md [hunk 2]
```diff
@@ -4,84 +4,89 @@ xhgui
...
+   the indexes, as in the following:
 
-    ```
-    cd path/to/xhgui
-    php install.php
-    ```
+   ```
+   $ mongo
+   > use xhprof
+   > db.results.ensureIndex( { 'meta.SERVER.REQUEST_TIME' : -1 } )
```
Comment (schmalliso) [inline README.md]: MongoDB 3.0 changed `ensureIndex` to `createIndex`. I'm not sure if that's something you care about.

File: README.md [hunk 3]
```diff
@@ -4,84 +4,89 @@ xhgui
...
-   [Uprofiler](https://github.com/FriendsOfPHP/uprofiler) to actually profile the data
- * [MongoDB PHP](http://pecl.php.net/package/mongo) MongoDB PHP extension
- * [MongoDB](http://www.mongodb.org/) MongoDB Itself
- * [mcrypt] (http://php.net/manual/en/book.mcrypt.php) PHP must be configured
-   with mcrypt (which is a dependency of Slim)
- * [dom] (http://php.net/manual/en/book.dom.php) If you are running the tests
-   you'll need the DOM extension (which is a dependency of PHPUnit)
+   [Uprofiler](https://github.com/FriendsOfPHP/uprofiler) to actually profile the data.
+ * [MongoDB PHP](http://pecl.php.net/package/mongo) MongoDB PHP driver.
+   XHGui requires verison 1.3.0 or later.
```
Comment (schmalliso) [inline README.md]: There is a new PHP driver, which is significantly different. Not sure if you'll be able to work with it, if not, a note should be noted here. The numbering scheme also changed, which is confusing. The new one is like, 1.0.0.0 or something.

Comment (markstory) [inline README.md]: If the driver is very different, it is unlikely that we'll be able to be compatible with both the old and new one :cry:

File: README.md (hunk 2 above)
Comment (markstory) [inline README.md]: We could mention it. I'm sure someone will get very confused running a new version of MongoDB.

File: README.md (hunk 3 above)
Comment (schmalliso) [inline README.md]: I suspect it's backward compatible, since the drivers team is pretty pro-that. BUT, worth double checking :)

**Comment by schmalliso** [comment]
Thanks, @markstory!

**Comment by markstory** [comment]
And thank you :dancer:

**Pull request merged**